README.md

```diff
@@ -0,0 +1,67 @@
+CMSIS-SVD Repository
+====================
+
+This repository provides a convenient place to access and aggregate CMSIS-SVD hardware descriptions from multiple sources.
```

Comment:
I could also see this repo as potentially being the place where scripts to "fetch" some SVDs (perhaps exclusively for populating/updating the contents of this repo) may also be stored.  Probably doesn't require any update until such time that such things exist.

Reply:
Hi, 

@BenBE I think it is beter that fetch and update scripts come here, since they are not related to cmsis-svd file format, and for pratical reasons it is easier to have these scripts here.

@posborne ok for me to put here scripts to fetch/update SVDs content of this repository. The README is updated to go this way.